modules/nf-core/plastid/make_wiggle/main.nf

```diff
@@ -0,0 +1,64 @@
+process PLASTID_MAKE_WIGGLE {
+    tag "$meta.id"
+    label "process_single"
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/plastid:0.6.1--py39had3e4b6_2':
+        'biocontainers/plastid:0.6.1--py39had3e4b6_2' }"
+
+    input:
+    tuple val(meta), path(bam), path(bam_index), path(p_offsets)
+    val min_length
+    val output_format
+    val fiveprime_variable
+
+    output:
+    tuple val(meta), path("*.{wig,bedgraph}"), emit: tracks
+    path "versions.yml", emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def extension = output_format == "bedgraph" ? "bedgraph" : "wig"
+    def args = task.ext.args ?: ""
+    if (fiveprime_variable) args += "--fiveprime_variable"
+    """
+    make_wiggle \\
+        --count_files "$bam" \\
+        --offset "$p_offsets" \\
+        --min_length "$min_length" \\
+        --output_format "$output_format" \\
+        --verbose \\
+        -o "$prefix" \\
+        $args
+
+    sed -i '/^##/d' *.wig # remove variable comment header
```

Comment:
Let's not edit command output. If you're doing this for the snapshot purposes there are other workarounds (like checking for content rather than a full snapshot)